tests/test_parser_simple.py

```diff
@@ -0,0 +1,794 @@
+from tests import common
+import unittest
+
+
+class TestSimpleParser(unittest.TestCase):
+    """
+        Test single line gherkin parser functionality of romaine's core.
+    """
+    def setUp(self):
```

Comment:
See other response. I'm happy to remove these if you prefer. I think they're currently in all of the tests, so they should at least be consistent, but if we'd prefer for them to consistently be absent I can rip them out.